pkg/streamingpromql/planning.go

```diff
@@ -170,6 +177,8 @@ func (p *QueryPlanner) NewQueryPlan(ctx context.Context, qs string, timeRange ty
 		return nil, err
 	}
 
+	spanLogger.DebugLog("msg", "AST optimisation passes complete", "expression", expr)
```

Comment:
Nits. Shouldn't this be "completed" to be consistent with other spanloggers too??
```suggestion
	spanLogger.DebugLog("msg", "AST optimisation passes completed", "expression", expr)
```